AndroidAssignment/app/src/main/java/org/sopt/androidassignment/HomeActivity.kt

```diff
@@ -0,0 +1,39 @@
+package org.sopt.androidassignment
+
+import android.content.ActivityNotFoundException
+import android.content.Intent
+import android.content.Intent.createChooser
+import android.net.Uri
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import android.widget.Toast
+import org.sopt.androidassignment.databinding.ActivityHomeBinding
+
+
+class HomeActivity : AppCompatActivity() {
+
+    private lateinit var binding : ActivityHomeBinding
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+
+        binding = ActivityHomeBinding.inflate(layoutInflater)
+        val address : Intent = Uri.parse("https://github.com/briandr97").let{webpage->Intent(Intent.ACTION_VIEW, webpage)}
```

Comment:
코틀린에서는 타입추론 기능을 지원하기 때문에
```suggestion
                val address = Uri.parse("https://github.com/briandr97").let{webpage->Intent(Intent.ACTION_VIEW, webpage)}
```
이런식으로 작성하셔도 괜찮습니다~!!

Reply:
그리고 혹시 스코프함수인 let을 쓰신 이유가 있을까요? 쓰지않아도 괜찮을 것 같다는 생각이 드네요~!!